### DISCUSSION TITLE
How should I make favicons for my Pelican Website?

### DISCUSSION THREAD

**Discussion by DragonSerW** [getpelican/pelican#3196] [Q&A]
Hello guys! :-))
32x32 pixels favicon.ico for my website is currently made by me from the image.svg: https://openclipart.org/detail/301994/yellow-paint-splat.
How should I resize that very .svg to meet the requirements of modern browsers?
And where should I place that Favicons? :-))

**Comment by virtadpt** [comment]
That is less a Pelican-software question and more a Pelican-theme question.  You'd add it to your site theme template, usually in the base.html file.  An example: https://github.com/virtadpt/brutalist/blob/main/templates/base.html

**Comment by lioman** [comment]
Put your `favocon.ico` into a folder in `content` e.g. `static`
Adapt your `pelicanconf.py` like this: 
```python
STATIC_PATHS = [
    …
    "static/favicon.ico",
]

EXTRA_PATH_METADATA = {
    …
    "static/favicon.ico": {"path": "favicon.ico"},
}
```
on build your icon will be put. to top folder and browsers will load this automatically, if you do not have specified this in your theme otherwise
